src/main/java/com/vesoft/nebula/jdbc/NebulaAbstractPreparedStatement.java

```diff
@@ -58,7 +61,9 @@ protected void insertParameter(int parameterIndex, Object obj) throws SQLExcepti
 
     @Override
     public ResultSetMetaData getMetaData() throws SQLException {
-        return this.getMetaData();
+
+        ResultSet currentResultSet = this.currentResultSet;
```

Comment:
`currentResultSet` is a protected member from its father, so this assignment is unnecessary.